frontend/src/routes/manage/Video/Access.tsx

```diff
@@ -53,7 +57,11 @@ const AclPage: React.FC<AclPageProps> = ({ event, data }) => {
         <Breadcrumbs path={breadcrumbs} tail={t("manage.my-videos.acl.title")} />
         <PageTitle title={t("manage.my-videos.acl.title")} />
         {event.hostRealms.length < 1 && <UnlistedNote />}
-        <AccessUI {...{ event, knownRoles }} />
+        <br />
+        {CONFIG.allowAclEdit
+            ? <AccessUI {...{ event, knownRoles }} />
+            : <Card kind="info">ACL editing is disabled.</Card>
```

Comment:
Hardcoded string? Should be a translation string, right? Even if most users shouldn't see this at all.